modules/test/playwright/tests/headless-builder-web/apiExplorer.spec.ts

```diff
@@ -48,9 +51,8 @@ const singleElementIdEndpoint = {
 test('can see filter and sort parameters for collection endpoints', async ({
 	apiExplorerPage,
 	apiHelpers,
-	page,
 }) => {
-	await apiHelpers.objectEntry.postObjectEntry(
+	const collectionEndpoint = await apiHelpers.objectEntry.postObjectEntry(
```

Comment:
This is an application, as you are creating an application with the application data and with an endpoint. Please rename it to something with more sense, like `const apiApplication'

Reply:
see https://github.com/liferay-headless/liferay-portal/pull/2322/commits/8c36851645a50ce9f9faf7ceb1445fb98970bea7